parler/tests/test_query_count.py

```diff
@@ -1,12 +1,19 @@
 import datetime as dt
 
+import django
 from django.core.cache import cache
 from django.utils import translation
 from django.utils.timezone import now
 from parler import appsettings
 
 from .utils import AppTestCase, override_parler_settings
-from .testapp.models import SimpleModel, DateTimeModel
+from .testapp.models import SimpleModel, SimpleLightModel, DateTimeModel
+
+try:
+    from unittest import skipIf
+except ImportError:
+    # python<2.7
+    from django.utils.unittest import skipIf
```

Comment:
Python 2.6 compatibility is not required. It's EOL and obsolete and parler doesn't support it anymore